.gitignore

```diff
@@ -55,8 +55,7 @@ pmd.bat
 .project
 .settings/
 .vscode/
-*/.project
-*/.settings
-*/.classpath
```

Comment:
is this change intentional ?

Reply:
yes it is. I'm trying to set up my eclipse and I'm seeing some files which should not be tracked (`*/.` covers only subdirectories, not files in root git dir)

Below is example without my changes in `.gitignore` file:
```
# git status --short

# touch .classpath
# touch opengrok-indexer/.classpath
# git status --short
?? .classpath
```
`.classpath` file is not shown in subdirectory, but it is visible in root directory.

BTW [.gitignore man page](https://git-scm.com/docs/gitignore):
>If the pattern does not contain a slash /, Git treats it as a shell glob pattern and checks for a match 
>against the pathname relative to the location of the .gitignore file (relative to the toplevel of the work tree 
>if not from a .gitignore file).